index.js

```diff
@@ -12,8 +12,44 @@ var unlink = RSVP.denodeify(fs.unlink);
 var tmpDir = require('os').tmpdir();
 var debug = require('debug')('async-disk-cache');
 var zlib = require('zlib');
+var heimdall = require('heimdalljs');
 
 var CacheEntry = require('./lib/cache-entry');
+var Metric = require('./lib/metric');
+
+heimdall.registerMonitor('async-disk-cache', function AsyncDiskCacheSchema() {});
```

Comment:
We need to guard against duplicate monitors (it throws a very annoying error).  The simplest way to trigger this would be to `npm link async-disk-cache` into a `ember new foo` app, then `npm link ember-cli`, then `npm link async-disk-cache` inside your `ember-cli` working directory.  You will have this monitor registered twice, which will go :boom:.

heimdall-fs-monitor works around this (in a somewhat convoluted way): https://github.com/heimdalljs/heimdall-fs-monitor/blob/master/index.js